instrumentation/rack/Gemfile

```diff
@@ -7,8 +7,8 @@
 source 'https://rubygems.org'
 
 gemspec
-
 group :test do
   gem 'opentelemetry-instrumentation-base', path: '../base'
+  gem 'opentelemetry-sdk-experimental'
```

Comment:
Do we need this here _and_ in the gemspec as a dev dependency?

Reply:
I thought so, but re-testing just now it doesn't seem necessary 🤔